src/keyfile.c

```diff
@@ -1287,13 +1285,6 @@ void configuration_apply_settings(void)
 	}
 	g_free(scribble_text);
 
-	/* set the position of the hpaned and vpaned */
-	if (prefs.save_winpos)
-	{
-		gtk_paned_set_position(GTK_PANED(ui_lookup_widget(main_widgets.window, "hpaned1")), hpan_position);
-		gtk_paned_set_position(GTK_PANED(ui_lookup_widget(main_widgets.window, "vpaned1")), vpan_position);
-	}
-
```

Comment:
I believe the only reason why it was here was that hpan_position and vpan_position were read in this file and accessible only here.